Makefile

```diff
@@ -190,6 +190,21 @@ ifeq ($(TARGET_PLATFORM), openshift)
 else
 	kubectl create ns operator || true
 	kubectl apply -f tests/operator-group.yaml
+ifneq ($(http_proxy),)
```

Comment:
can we avoid those if conditions? maybe with the "<http_proxy>" in quotes?

Reply:
Good point, I corrected that.